views/dataview.py

```diff
@@ -7,4 +9,4 @@ def get_layout(session_id):
     Keywords arguments:
     session_id -- ID of the current session
     """
-    return data_table.get_component(session_id)
+    return dcc.Loading(data_table.get_component(session_id))
```

Comment:
Perhaps it is better if the function `data_table.get_component` returns the loading component wrapper, does it make a difference?

Reply:
Good idea, I've now changed it